app/javascript/application.js

```diff
@@ -0,0 +1,29 @@
+// Entry point for the build script in your package.json
+// import "@hotwired/turbo-rails"
+import "./controllers"
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {
+  createBrowserRouter,
+  RouterProvider,
+} from "react-router-dom";
+import "./index.css";
+import {Provider} from "react-redux";
+import {store} from "./store.js";	
+
+import Greeting from "./Greeting.js";
+
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: <Greeting/>,
+  },
+]);
```

Comment:
- [ ] Kindly make sure to create an `App` component in which you can add the router configurations